src/rgw/rgw_common.h

```diff
@@ -786,8 +791,10 @@ struct RGWBucketInfo
   // Represents the shard number for blind bucket.
   const static uint32_t NUM_SHARDS_BLIND_BUCKET;
 
+  RGWBucketIndexType index_type;
+
   void encode(bufferlist& bl) const {
-     ENCODE_START(11, 4, bl);
+     ENCODE_START(12, 4, bl);
```

Comment:
@oritwas it should be v15 for index type. Need to keep the same versions and make sure it encodes/decodes correctly in other versions.